### ISSUE TITLE
1x4 and 2x4 mixer bugs

### ISSUE THREAD

**Issue by blackearth2014** [mschack/VCV-Rack-Plugins#18]
msHack,

Sorry to do this to you. The 1x4 mixer meters doesn't work. The meters on the 2x4 mixer are not functioning properly.  Some meters only has the right meter pumping when there is only a left channel connection. Some channel doesn't work and L/R channel meter errors.

**Comment by mschack** [comment]
I haven't figured that one out yet.  It happens randomly.  Restarting VCV Rack sometimes fixes the issue.

**Comment by mschack** [comment]
I think I fixed this one.

**Comment by blackearth2014** [comment]
The ping pong delay is now not working for some reason. The Aux sends works but the delay is not processing or any audio is passing through.

**Comment by mschack** [comment]
lol, what the heck I only replaced the button...

**Comment by blackearth2014** [comment]
maybe the button meta was updated in the code

**Comment by mschack** [comment]
It is my own button.  I did test this too, not sure what happened.  Working on it.

**Comment by blackearth2014** [comment]
I feel dumb. I just found out that the 6x32 trigger sequencer 3 color button knobs controls the levels of your drum synth. I didn't undertand the levels at first. I was always using the Trig output port to the levels port of the drum synth or using it as a 2 note trigger to the 3 OSC synth.

**Comment by mschack** [comment]
yeah, works nice using it to control a filter or output level.  Put it into Bi Directional mode and control pan with it.

Pong is fixed and updated...

**Comment by blackearth2014** [comment]
Thanks. By the way, will you be able to use the same kind of chain linking with the mixer like you did with the master clock. When chaining , do you have to using two cables for audio out where as you can implement that in the code using one cable linking.

**Comment by mschack** [comment]
No I don't think I could, just because I would not be able to send two floating point signals L, and R, every system step (each step is basically the sample rate so 44100 times per second or whatever).  The clock was easy because I don't have to send two separate signals every time, it just sends the tick when it occurs (-1) or the BPM (40-220) so it is easy to distinguish between them.
